kamailio/trunks/config/kamailio.cfg

```diff
@@ -2112,6 +2117,27 @@ event_route[xhttp:request] {
     exit;
 }
 
+route[END_COMPANY_CALLS] {
+    sql_query("cb", "SELECT B.id, C.id FROM Companies C JOIN Brands B ON B.id=C.brandId", "ra");
```

Comment:
Unless you actually need something from Brands table, you can get Brand id from Companies.

    SELECT brandId, id FROM Companies

Reply:
So true, I need holidays